RecoTracker/Configuration/python/customiseForQuadrupletsByCellularAutomaton.py

```diff
@@ -1,40 +1,90 @@
 import FWCore.ParameterSet.Config as cms
 
-def customiseForQuadrupletsByCellularAutomaton(process):
-    for module in process._Process__producers.values():
-        if not hasattr(module, "OrderedHitsFactoryPSet"):
-            continue
-	pset = getattr(module, "OrderedHitsFactoryPSet")
-        if not hasattr(pset, "ComponentName"):
-	    continue
-	if not (pset.ComponentName == "CombinedHitQuadrupletGenerator"):
-	    continue    
-        # Adjust seeding layers
-        seedingLayersName = module.OrderedHitsFactoryPSet.SeedingLayers.getModuleLabel()
+_CAParameters = dict(
+    maxChi2 = dict(
+        pt1    = 0.8, pt2    = 2,
+        value1 = 200, value2 = 100,
+        enabled = True,
+    ),
+    useBendingCorrection = True,
+    fitFastCircle = True,
+    fitFastCircleChi2Cut = True,
+    CAThetaCut = 0.00125,
+    CAPhiCut = 0.1,
+    CAHardPtCut = 0,
+)
+
+def customiseLegacySeeding(module):
+    pset = getattr(module, "OrderedHitsFactoryPSet")
```

Comment:
> Full migration better be done sooner than later.

I agree, and I'm working on that (but I need one episode with PixelTrackProducer before that). FWIW, I already have a private version of a function for `customizeHLTforCMSSW.py` migrating everything in this PR.

I decided to not include it yet because HLT uses some components that are not used in RECO that need to be migrated, and I'd rather have one function in `customizeHLTforCMSSW.py` doing the migration than many (and I wanted to get these developments under review).